packages/browser/src/client.ts

```diff
@@ -85,19 +85,23 @@ export class BrowserClient extends Client<BrowserClientOptions> {
     }
 
     this.on('postprocessEvent', event => {
-      addAutoIpAddressToUser(event);
+      if (this._options.sendDefaultPii) {
```

Comment:
hmm, I wonder if we can just conditionally not add both of these hooks at all here if this is not on? 🤔 a bit simpler, you loose the ability to flip this option later, but is this something we want to/need to support?

Reply:
uh totally. Mutating options is kinda not something we intentionally support. Whenever it works it is usually by accident.